manifest-build-tools/lib/manifest.py

```diff
@@ -0,0 +1,253 @@
+# Copyright 2016, EMC, Inc.
+
+"""
+Module to abstract a manifest file
+"""
+import json
+import os
+import re
+import sys
+import datetime
+import config
+
+from gitbits import GitBit
+
+
+class Manifest(object):
+    def __init__(self, file_path, git_credentials = None):
+        """
+        __build_name - Refer to the field "build-name" in manifest file.
+        __repositories - Refer to the field "repositories" in manifest file.
+        __downstream_jobs - Refer to the field "downstream-jobs" in manifest file.
+        __file_path - The file path of the manifest file
+        __name - The file name of the manifest file
+        __manifest -The content of the manifest file
+        __changed - If manifest is changed, be True; The default value is False
+        __git_credentials  - URL, credentials pair for the access to github repos
+        gitbit - Class instance of gitbit
+        """
+
+        self._build_name = None
+        self._build_requirements = None
+        self._repositories = []
+        self._downstream_jobs = []
+        self._file_path = file_path
+        self._name = file_path.split('/')[-1]
+        self._manifest = None
+        self._changed = False
```

Comment:
Oh, yes.
The member is used in updating manifest.
I will submit another commit to remove it.